### PR TITLE
Fix for failing RPC generator tests

### LINKED ISSUE #1845: RPC generator tests failing
### Bug Report
RPC generator tests are not successful

##### Reproduction Steps
1. run RPC generator tests (runner.py)

##### Expected Behavior
Tests done successfully

##### Observed Behavior
Tests not done successfully

### PR DESCRIPTION
**Pull request by FrankElias77**
Fixes #1845 

### Risk
This PR makes **no** API changes.

### Testing Plan
- [x] I have verified that I have not introduced new warnings in this PR (or explain why below)
- [x] I have run the unit tests with this PR
- [ ] I have tested this PR against Core and verified behavior (if applicable, if not applicable, explain why below).

#### Unit Tests
N/A

#### Core Tests
N/A

Core version / branch / commit hash / module tested against: N/A
HMI name / version / branch / commit hash / module tested against: N/A

### Summary
RPC generator tests were failing at some steps for not being updated after doing changes on the generator.

##### Breaking Changes
* N/A

##### Enhancements
* N/A

##### Bug Fixes
* RPC generator tests were failing at some steps for not being updated after doing changes on the generator.

### Tasks Remaining:
N/A

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Review by joeljfischer — CHANGES_REQUESTED** [review]
There's one more issue:

1. If the CI tests fail, the RPC Generator Tests still say they succeed.

File: generator/test/test_enums.py [hunk 1]
```diff
@@ -43,12 +43,17 @@ def test_FunctionID(self):
         expected = OrderedDict()
         expected['origin'] = 'FunctionID'
         expected['name'] = 'SDLFunctionID'
-        expected['imports'] = {'.h': {'SDLEnum'}, '.m': {'SDLEnum'}}
+        expected['imports'] = {
+            '.h': {
+                'SDLEnum'},
+            '.m': [
+                'SDLEnum']}
```
[the remark this answers is not in the document]
Comment (FrankElias77) [reply generator/test/test_enums.py:50]: This expansion doesn't change anything related to execution or performance, just trying to format to make it similar to other classes. Will modify it.

File: generator/test/test_enums.py [hunk 2]
```diff
@@ -43,12 +43,15 @@ def test_FunctionID(self):
         expected = OrderedDict()
         expected['origin'] = 'FunctionID'
         expected['name'] = 'SDLFunctionID'
-        expected['imports'] = {'.h': {'SDLEnum'}, '.m': {'SDLEnum'}}
+        expected['imports'] = {
+            '.h': {'SDLEnum'},
+            '.m': ['SDLEnum']}
```
Comment (joeljfischer) [inline generator/test/test_enums.py:48]:
```suggestion
        expected['imports'] = {
            '.h': {'SDLEnum'},
            '.m': ['SDLEnum']
        }
```

File: generator/test/test_enums.py [hunk 3]
```diff
@@ -69,14 +72,19 @@ def test_TextFieldName(self):
         expected = OrderedDict()
         expected['origin'] = 'TextFieldName'
         expected['name'] = 'SDLTextFieldName'
-        expected['imports'] = {'.h': {'SDLEnum'}, '.m': {'SDLEnum'}}
+        expected['imports'] = {
+            '.h': {
+                'SDLEnum'},
+            '.m': [
+                'SDLEnum']}
```
Comment (joeljfischer) [inline generator/test/test_enums.py:79]:
```suggestion
        expected['imports'] = {
            '.h': {'SDLEnum'},
            '.m': ['SDLEnum']
        }
```

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -78,3 +78,22 @@ jobs:
...
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v2.3.1
+              with:
+                  submodules: true
+            - name: Current Directory
+              run: ls
+            - name: Install dependencies
+              run: |
+                python3 -m pip install -r generator/requirements.txt
```
Comment (joeljfischer) [inline .github/workflows/test.yml:96]: Do you need to do the pipe or can you just put the bottom line on the same line as the `run` command like other test in the file above

**Review by joeljfischer — CHANGES_REQUESTED** [review]
A few more small code-style issues

File: .github/workflows/test.yml (hunk 4 above)
Comment (FrankElias77) [reply .github/workflows/test.yml]: I can remove the pipe. It was used before to combine commands, now there is no need to it.

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -78,3 +78,20 @@ jobs:
...
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v2.3.1
+              with:
+                  submodules: true
+            - name: Current Directory
+              run: ls
```
Comment (joeljfischer) [inline .github/workflows/test.yml:93]: Is this command necessary or was it only for debug?

**Review by joeljfischer — CHANGES_REQUESTED** [review]
Just one more

File: .github/workflows/test.yml (hunk 5 above)
Comment (FrankElias77) [reply .github/workflows/test.yml:93]: It is not necessary, i added it for debug

**Review by joeljfischer — APPROVED** [review]

**Pull request merged**